### PR TITLE
Issue #223: Add our logo

### PR DESCRIPTION
**Pull request by pascalchevrel**
- add the logo in svg format and make it scale via CSS for screen widths
- add a svg favicon for Firefox and a 16x16 simple png one for other browsers
- store all the logos in the web/img/logo/ folder
- Credit Elio Qoshi and the Mozilla Community Design team

resources

### CODE REVIEW

**Comment by pascalchevrel** [comment]
@flodolo and @TheoChevalier what do you think?

**Comment by flodolo** [comment]
I like it. Some notes:
* Should we get rid of ´$title´ in `dispatcher.php`? I didn't check in full details, but I don't think it's used anywhere.
* Should we put a link to the home page on the logo? I definitely used the old title to go back.

**Comment by flodolo** [comment]
Since we get rid of the "Transvision" fixed badge at the top, we can recover some top margin in the menu (if we want).

**Comment by pascalchevrel** [comment]
I added the link back and removed $title

File: app/views/credits.php [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 <p>Transvision is available on <a href="https://github.com/mozfr/transvision/">GitHub</a></p>
 
+<p>The Transvision logo was created by Elio Qoshi from the <a href="https://github.com/mozilla/Community-Design">Mozilla Community Design team</a></p>
```
Comment (flodolo) [inline app/views/credits.php]: Looks like this sentence would need a period at the end (also the one before).

**Pull request merged**